spec/helpers/checkouts_helper_spec.rb

```diff
@@ -51,20 +51,40 @@
 
     context 'when an item is not recalled' do
       it 'renders the right html' do
-        expect(content).to have_css('span', text: 'November 14, 2019 11:59pm')
+        expect(content).to have_css('span', text: 'November 14, 2019')
       end
     end
 
     context 'when an item is recalled' do
       before do
         allow(checkout).to receive_messages(
           recalled?: true,
-          recall_due_date: Time.zone.parse('2019-11-10T23:59:00-05:00')
+          recall_due_date: Time.zone.parse('2019-11-10T22:30:00-05:00')
         )
       end
 
       it 'renders the right html' do
-        expect(content).to include('<span>Recalled<br>November 10, 2019 11:59pm<br>November 14, 2019 11:59pm</span>')
+        expect(content).to include('<span>Recalled<br>November 10, 2019 10:30pm<br>November 14, 2019</span>')
```

Comment:
Out of curiosity, I wonder why you are suing `include` here and above you used `have_css` for seemingly the same kinda check. Just kinda random thing? I don't know if one is less expensive than the other tbh.

Reply:
hmmm no specific reason that I can remember of, I will try it with `have_css`, good point

Reply:
yeah idunno, dont feel like you have to, it's fine the way it is - it will definitely work either way.

Reply:
updated, see latest